### PR TITLE
I feel pretty

### PR DESCRIPTION
**Pull request by ironng**
Configures options for ~Prettier and prettifies js files. 💅~ eslint.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "singleQuote": true,
+  "overrides": [
+    {
+      "files": "*.scss",
+      "options": {
+        "tabWidth": 4
+      }
+    }
+  ]
+}
```
Comment (Shadowfiend) [inline .prettierrc:15]: Can we actually pull this whole config from our common eslint config at https://github.com/keep-network/eslint-config-keep ? https://github.com/keep-network/tbtc/blob/1149b2f0e1bf6d34c4f9897835699c406fdfe3fc/solidity/package.json#L54-L56 and https://github.com/keep-network/keep-ecdsa/blob/master/solidity/.eslintrc show how this looks in practice I believe.

Trying to keep our styles similar across the board. Tbh I want to make the eslintrc's strictly have `extends` lines, and I'm not sure we need to explicitly extend `prettier` as `eslint-config-keep` should already extend it.

Comment (ironng) [reply .prettierrc:15]: Oooh even better! Sorry I didn't know about this. I agree, the code should look as similar across the board as possible!

**Comment by ironng** [comment]
@Shadowfiend Moved this back to a draft since this still needs tinkering. 

>Tbh I want to make the eslintrcs strictly have extends lines

There needs to be a bit of customization on top of the keep config to handle React stuff...would it be better if we created a keep config specific to React instead of customizing here? (Also, what are your thoughts on using [eslint-plugin-react](https://www.npmjs.com/package/eslint-plugin-react) in general)?

I'm thinking about reverting the changes from 89870de so this doesn't block other work, since this might not be top priority.

**Comment by Shadowfiend** [comment]
> would it be better if we created a keep config specific to React instead of customizing here

I would actually love this but we might could wait a cycle or two. Certainly between this, tBTC site, Keep dApp, and tBtC dApp, we've got plenty of opportunity for unifying our React style as well. Tom also discovered [this sick feature](https://www.flowdock.com/app/cardforcoin/fold-flow/messages/316307) to manage the commits we use to apply the big initial formatting hit.

**Comment by ironng** [comment]
> Tom also discovered this sick feature 

😍 Oh yes, that would be amazing. Waiting to make a standardized React style config also sounds good given all the projects!

**Comment by Shadowfiend** [comment]
I'm down to roll with this if you want to hit the codebase... Only thing I'll note is it'd be cool to have a GitHub Action that runs eslint and yells if you missed it, but I don't think that's blocking.

**Comment by Shadowfiend** [comment]
Oh, and a precommit hook :grimacing:

**Comment by ironng** [comment]
Will add the action + hook! Going to wait for #177 to go through before hitting the codebase to avoid conflicts.

File: .eslintrc [hunk 2]
```diff
@@ -8,6 +8,10 @@
     "ecmaVersion": 2017,
     "sourceType": "module"
   },
+  "rules": {
+    // Fixes eslint no-invalid-this error for fat arrow class methods
+    "no-invalid-this": 0
```
Comment (Shadowfiend) [inline .eslintrc:13]: Where was this flagging? Wondering if it's something we should turn off for now but look to fix + turn back on.

Comment (ironng) [reply .eslintrc:13]: It was flagging the usage of `this` in the `onChange`, `onClick` and `onKeyUp` in the `EmailForm` component. 
https://github.com/keep-network/website/blob/5afdd75c1a8072fa7615283e84390e1535d654ce/src/components/EmailForm.js#L53, https://github.com/keep-network/website/blob/5afdd75c1a8072fa7615283e84390e1535d654ce/src/components/EmailForm.js#L75
https://github.com/keep-network/website/blob/5afdd75c1a8072fa7615283e84390e1535d654ce/src/components/EmailForm.js#L80

I think it's nice to leave on so we can use arrow functions for instance methods on a class, but maybe there's a better way to make this pass the linter. However, I'd like to refactor this component to use react hooks instead anyhow.

Comment (Shadowfiend) [reply .eslintrc:13]: That refactoring is probably the Ideal™ fix. Cool cool!

**Pull request merged**